tests/test_fock.py

```diff
@@ -611,17 +611,14 @@ def circuit(r, phi):
         expected = np.exp(2 * r) * np.sin(phi) ** 2 + np.exp(-2 * r) * np.cos(phi) ** 2
         assert np.allclose(var, expected, atol=tol, rtol=0)
 
-        # circuit jacobians
-        gradA = circuit.jacobian([r, phi], method="A")
-        gradF = circuit.jacobian([r, phi], method="F")
+        grad = qml.jacobian(circuit)(r, phi)
```

Comment:
Two mutually exclusive suggestions, not sure which one is the best one:

* you could make the replacement:

  ```python
  gradA = circuit.qtape.jacobian([r, phi], method="analytic")
  gradF = circuit.qtape.jacobian([r, phi], method="numeric")
  ```

* Alternatively, you can create two QNodes:

  ```python
  qnodeA = qml.QNode(circuit, dev, diff_method="parameter-shift")
  qnodeF = qml.QNode(circuit, dev, diff_method="finite-diff")
  gradA = qml.jacobian(qnodeA)(r, phi)
  gradF = qml.jacobian(qnodeF)(r, phi)
  ```

I'm not sure which is preferable 🤔 Maybe the first? I'm just hesitant to remove the finite difference check.

Reply:
Thanks @josh146! I briefly thought about the first option as well, but decided that it worked well enough just using `qml.jacobian`. But you're probably correct. Still WIP though, so I'll look into it more tomorrow. 🙂